### ISSUE TITLE
NullPointerException in org.terasology.logic.delay.DelayedActionSystem.invokeDelayedOperations

### ISSUE THREAD

**Issue by fnuecke** [MovingBlocks/Terasology#2109]
Game log: http://pastebin.com/A4Bxb26w
Init log: http://pastebin.com/WrFbMvjM

I hope this isn't a duplicate, didn't immediately find anything obviously related.

This was a fresh world of TTA (after exiting the game and reloading into it once after ~10 minutes of running around). Possibly noteworthy, as evident in the log (after figuring out the console) I set the world time back to 0, then to 0.5. After that I just dug some dirt, made the first tools, and then the NPE happened.

If you need any more info, let me know!

**Comment by Cervator** [comment]
I don't think this is a dupe and I do think it is a legit bug. It looks familiar to me as a thing I've seen once in a while, but not consistently enough to have any clues how to replicate or fix it.

With some more eyes on it maybe we can narrow this bug down some and stomp it :)

There are a few rough edges around the corners of the entity system. We've stabilized most of the common stuff but know of a few edge cases that could be improved. @immortius is working on that occasionally as the author of the system and @MarcinSc finds the occasional bug as he gets deep into the workings of the systems with his more complex logic. @flo also has good experience with the system and has tweaked at improvements at times.
